lamp.py

```diff
@@ -0,0 +1,53 @@
+"""Class to represent dali lamps"""
+import json
+from consts import (
+    __author__,
+    __version__,
+    __email__,
+    MQTT_STATE_TOPIC,
+    MQTT_COMMAND_TOPIC,
+    MQTT_PAYLOAD_OFF,
+    MQTT_BRIGHTNESS_STATE_TOPIC,
+    MQTT_BRIGHTNESS_COMMAND_TOPIC,
+    MQTT_DALI2MQTT_STATUS,
+    MQTT_AVAILABLE,
+    MQTT_NOT_AVAILABLE,
+)
+
+
+class Lamp:
+    def __init__(self, address, min_physical_level, min_level, max_level):
+        self.address = address
```

Comment:
address should be short_address (because there is the full address we don't use)

Reply:
Done